src/cljs/ataru/virkailija/editor/components/text_component.cljs

```diff
@@ -115,31 +210,38 @@
       (reset! show-followups (vec (replicate option-count true))))
     (when (< 0 option-count)
       (let [option-index 0
-            followups    (nth followups option-index)]
+            followups    (nth followups option-index)
+            option       (nth options option-index)]
         [:div.editor-form__text-field-option-followups-wrapper
-         [followup-question-overlay option-index followups path show-followups]]))))
+         (when (:condition option)
+           [:div.editor-form__text-field-option-followups-header
+            [text-field-option-condition (:condition option) path]
+            [followup-question/followup-question option-index followups show-followups]])
+         [followup-question/followup-question-overlay option-index followups path show-followups]]))))
 
 (defn- text-field-has-an-option [_ _ _ _]
   (let [id (util/new-uuid)]
     (fn [value followups path component-locked?]
-      (let [option-index 0
-            has-options? (not (empty? (:options value)))
-            repeatable?  (-> value :params :repeatable boolean)
-            disabled?    (or component-locked?
-                             (not (empty? (first followups)))
-                             repeatable?)]
+      (let [option-index               0
+            options-without-condition? (not (empty? (remove :condition (:options value))))
```

Comment:
Mä kirjoittaisin nämä threading-makrolla, tyyliin `(->> value :options (remove :condition) empty? not)` ja minimoisin sulkeiden määrän.